packages/datepicker/datepicker.scss

```diff
@@ -0,0 +1,70 @@
+@import "~@fremtind/jkl-core/build/scss/variables/spacing.scss";
+@import "~@fremtind/jkl-core/build/scss/variables/colors.scss";
+@import "~@fremtind/jkl-core/build/scss/typography/paragraphs.scss";
+@import "~@fremtind/jkl-core/build/scss/functions.scss";
+@import "~@fremtind/jkl-text-input/text-input.scss";
+
+$calendar-width: rem(402px);
+$calendar-box-size: rem(46px);
+
+.jkl-datepicker {
+    width: $calendar-width;
+    background-color: $snøhvit;
+
+    &__toggler {
+        border: none;
+        outline: none;
+        background-color: transparent;
+        text-align: left;
+        width: $calendar-width;
+    }
+    &__calendar {
+        display: flex;
+        justify-content: center;
+        flex-direction: column;
+        width: calc($calendar-width - $component-spacing--xl);
+        box-shadow: 0px rem(4px) rem(8px) rgba(0, 0, 0, 0.08);
+        padding: $component-spacing--xl;
+        &__header {
+            display: flex;
+            justify-content: center;
+            align-content: center;
+            &--year {
+                max-width: 30%;
+            }
+            &--month {
+                flex-grow: 2;
+            }
+        }
+
+        caption {
+            @extend .jkl-p;
```

Comment:
As above. This will include the style rules for paragraphs in duplicate for this component (since we import `paragraph.scss`). Optimally we would just give the element the `.jkl-p` class, but I guess that is not an option here. I still think we should try to get rid of the duplicate. Maybe make a paragraph `@mixin` instead?